src/System.Windows.Forms/src/System/Windows/Forms/Application.cs

```diff
@@ -276,6 +276,7 @@ public static DarkMode DefaultDarkMode
     ///  </para>
     /// </remarks>
 
+    [Experimental("WFO9000")]
```

Comment:
I very much encourage you to use [`UrlFormat`](https://learn.microsoft.com/dotnet/api/system.diagnostics.codeanalysis.experimentalattribute.urlformat?view=net-8.0#system-diagnostics-codeanalysis-experimentalattribute-urlformat) parameter as well
```suggestion
    [Experimental("WFO9001", UrlFormat = Experiments.SharedUrlFormat)]
```

Then follow the same procedures as you do for the obsoletions - aka.ms links, the doc page, etc. This'd make it easy for the team (from the maintenanace perspective) and for your users (from the consumption perspective).

Reply:
I will address this once I the Analyzer PR got merged. I want these IDs also defined as consts. That PR will take care of that.

Reply:
Done.